.github/workflows/linux_builds.yml

```diff
@@ -194,7 +194,8 @@ jobs:
         if: ${{ matrix.godot-cpp-test }}
         uses: actions/checkout@v3
         with:
-          repository: godotengine/godot-cpp
+          repository: mihe/godot-cpp
+          ref: error-macros
```

Comment:
This makes it pass CI but cannot be merged like this, so it shouldn't be part of the PR. The proper way is to merge first the change in godot-cpp, then re-check this PR.

Reply:
Failing CI shouldn't prevent it to be reviewed. 

> I was under the impression that me or the person merging this PR would drop that last commit just before merging.

Usually Akien does the merging, I'm not sure if he has a workflow to do this (and if so, if GitHub would consider this "merged" or just "closed"). I'm more concerned it might be approved and then merged including the commit by accident just because of an oversight.